client/components/paginate/PaginatorControlGroup.tsx

```diff
@@ -1,7 +1,12 @@
 import React from "react";
-import PropTypes from "prop-types";
 
-const PaginatorControlGroup = ({ callback, children, label }) => {
+interface PaginatorControlGroupProps {
+    callback: void;
```

Comment:
Another void function that maybe should be represented differently.

Reply:
Made change.